braindecode/classifier.py

```diff
@@ -268,3 +270,17 @@ def predict(self, X):
 
         """
         return self.predict_proba(X).argmax(1)
+
+    def predict_trials(self, X):
+        """Return trialwise predictions and targets.
+
+        Parameters
+        ----------
+        X : braindecode.datasets.BaseConcatDataset
+            A braindecode dataset to be predicted.
+
+        Returns
+        -------
+        trial_predictions, trial_labels: tuple(np.ndarray, np.ndarray)
+        """
+        return predict_trials(self.module, X)
```

Comment:
Maybe also check if self.cropped is True and if not raise error?

Reply:
Well, should there be an error? We always have trials no matter if we do cropped decoding or not.

Reply:
I think 
a) Right now code would fail without cropped decoding? There are a lot of assumptions inside coming from cropped decoding no? Or would it just run?
b) for trialwise decoding existing predict method would already give you trialwise predictions right? 

If that is correct either we assert we are in cropped mode (and maybe also add to name like predict_trials_from_cropped) or we just call regular predict function if not in cropped mode? We should check what happens atm when this function is called e.g. at end of trialwise decoding example

Reply:
I am not so much familiar with the internals of the cropped decoding to know about the assumptions.
In my opinion `predict_trials` should always work, no matter if cropped or trialwise decoding, since we always have trials.
For trialwise decoding the output will then just be the same as calling `predict`. For cropped decoding it will be different.